src/Microsoft.Azure.WebJobs/QueueAttribute.cs

```diff
@@ -26,7 +26,7 @@ namespace Microsoft.Azure.WebJobs
     [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1813:AvoidUnsealedAttributes")]
     [AttributeUsage(AttributeTargets.Parameter)]
     [DebuggerDisplay("{QueueName,nq}")]
-    public class QueueAttribute : Attribute
+    public class QueueAttribute : StorageAccountAttribute
```

Comment:
The value is to us as framework authors. We have a base attribute class that is extensible and makes more sense from a type perspective. StorageAccountAttribute is for specifying a storage account name value - that's why I named it the way I did. It was not designed as a base class - only to specify a single account name. StorageAttribute (abstract base class) is the base class for all storage binding attributes.

This won't be confusing for our users - the base class is abstract and they won't even think about it.